avocado/utils/multipath.py

```diff
@@ -161,6 +161,36 @@ def get_path_status(disk_path):
                     return(paths['dm_st'], paths['dev_st'], paths['chk_st'])
 
 
+def fail_path(path):
+    """
+    failing the individual paths
+    :param disk_path: disk path. Example: sda, sdb.
+    :return: True or False
+    """
+    cmd = 'multipathd -k"fail path %s"' % path
+    if process.system(cmd) == 0:
+        time.sleep(2)
+        path_stat = get_path_status(path)
+        if path_stat[0] == 'failed' or path_stat[2] == 'faulty':
```

Comment:
`and` or `or` here ? ie, should you check both are only one here ? Since you are checking both while reinstating the path